tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -337,6 +337,31 @@ stages:
         inputs:
           ArtifactName: onnxruntime_gpu
 
+      - script: |
+          'python3 -m pip install twine'
+        displayName: 'pip install twine'
+        condition: succeededOrFailed()
+        timeoutInMinutes: 20
+
+      - task: TwineAuthenticate@1
+        inputs:
+          pythonUploadServiceConnection: 'ortgpunightlytraining'
+        displayName: 'TwineAuthenticate ortgpunightlytraining'
+        condition: succeededOrFailed()
+        timeoutInMinutes: 20
+
+      - script: |
+          python3 -m twine upload --repository-url https://test.pypi.org/legacy/ --config-file $(PYPIRC_PATH) $(Build.ArtifactStagingDirectory)/dist/*.whl
+        displayName: 'python3 -m twine upload --repository testpypi $(Build.ArtifactStagingDirectory)/dist/*.whl'
```

Comment:
>testpypi [](start = 59, length = 8)

Do we consider using --pre vs test.pypi?

Reply:
changed to pypi repo. version is 1.7.0.dev202103133 as a pre-release